lib/administrate/search.rb

```diff
@@ -3,16 +3,24 @@
 
 module Administrate
   class Search
+    BLACKLISTED_WORDS = %w{destroy remove delete update create}
+
+    attr_reader :resolver, :term, :scope
+
     def initialize(resolver, term)
+      term ||= ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.